packages/core-react/src/components/player/Player.tsx

```diff
@@ -15,7 +20,11 @@ export type InternalPlayerProps = {
   _screenWidth: number | null;
 };
 
-export type PlayerProps<TElement, TPoster> = {
+export type PlayerProps<
+  TElement,
+  TPoster,
+  TPlaybackPolicyObject extends object,
```

Comment:
WDYT of having a default on this (and corresponding others)?
```suggestion
  TPlaybackPolicyObject extends object = object,
```

Reply:
Yeah there are defaults, they're just pushed higher up in the component tree (to save the headache of types not agreeing in a higher component)